### PR TITLE
Fix interventions amendments issues

### PR DESCRIPTION
**Pull request by daniel-tabarcea**
- fixed an issue with modified fields getting reset when
entering amendment mode
- removed add amendment button when interveiton has in_amendment: true
- added redirect to details when adding an amendment
- related to clubhouse issues 4405 4390 4398

### CODE REVIEW

**Comment by robertavram** [comment]
amendment permissions done on the backend

File: pmp/pages/page-interventions.html [hunk 1]
```diff
@@ -364,17 +364,21 @@
...
         }
         if (!this.tabAttached || this.listActive) {
           return;
         }
-        if (typeof newValue === 'undefined') {
+        if (typeof amendmentModeActive === 'undefined') {
           return;
         }
-        this.fire('set-amendment-mode', {active: newValue});
+        if (amendmentModeActive === true) {
```
Comment (acory) [inline pmp/pages/page-interventions.html]: Refreshing the Review&Sign page, while in amendment, redirects to details

**Comment by adi130987** [comment]
`removed add amendment button when interveiton has in_amendment: true` is not needed, permissions are changed from BE for this :)

File: pmp/app-elements/interventions/amendments/intervention-amendments.html [hunk 2]
```diff
@@ -207,6 +207,7 @@
           this._unlockInterventionDetailsFields();
           this.push('dataItems', data);
           this.set('amendActionInProgress', true);
+          this.fire('new-amendment-added');
```
Comment (adi130987) [inline pmp/app-elements/interventions/amendments/intervention-amendments.html]: Where is this event handled?

**Pull request merged**